civicrm.php

```diff
@@ -257,8 +257,8 @@ public static function singleton() {
         include_once CIVICRM_PLUGIN_DIR . 'wp-cli/wp-cli-civicrm.php';
       }
 
-      define('CIVICRM_IFRAME', self::$instance->is_iframe());
-      if (CIVICRM_IFRAME) {
+      if (self::$instance->is_iframe()) {
+        define('CIVICRM_IFRAME', 1);
```

Comment:
@mattwire I think the test failures might relate to this not always being defined any more

Reply:
Should be resolved now thankyou